Anlab.Mvc/Controllers/SystemController.cs

```diff
@@ -1,30 +1,34 @@
-using System;
-using AnlabMvc.Models.Roles;
-using AnlabMvc.Services;
-using Microsoft.AspNetCore.Authorization;
-using Microsoft.AspNetCore.Mvc;
-using System.Threading.Tasks;
-
-namespace AnlabMvc.Controllers
-{
-    [Authorize(Roles = RoleCodes.Admin)]
-    public class SystemController : ApplicationController
-    {
-        private readonly IDbInitializationService _dbInitializationService;
-
-        public SystemController(IDbInitializationService dbInitializationService)
-        {
-            _dbInitializationService = dbInitializationService;
-        }
-        public async Task<IActionResult> ResetDb()
-        {
-#if DEBUG
+using System;
+using AnlabMvc.Models.Roles;
+using AnlabMvc.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace AnlabMvc.Controllers
+{
+    [Authorize(Roles = RoleCodes.Admin)]
+    public class SystemController : ApplicationController
+    {
+        private readonly IDbInitializationService _dbInitializationService;
+
+        public SystemController(IDbInitializationService dbInitializationService)
+        {
+            _dbInitializationService = dbInitializationService;
+        }
+
+#if DEBUG
+        public async Task<IActionResult> ResetDb()
+        {
             await _dbInitializationService.RecreateAndInitialize();

```

Comment:
Actually, I'd say throw the exception here too. If we ever want to run the recreate, we should be very specific about doing it just in case we are running locally and pointing to prod.